platforms/core-runtime/launcher/src/integTest/groovy/org/gradle/launcher/daemon/DaemonToolchainCoexistWithCurrentOptionsIntegrationTest.groovy

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.launcher.daemon
+
+import org.gradle.integtests.fixtures.AvailableJavaHomes
+import org.gradle.integtests.fixtures.daemon.DaemonToolchainIntegrationSpec
+import org.gradle.internal.jvm.Jvm
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.IntegTestPreconditions
+
+@Requires(IntegTestPreconditions.NotEmbeddedExecutor)
+class DaemonToolchainCoexistWithCurrentOptionsIntegrationTest extends DaemonToolchainIntegrationSpec {
+
+    @Requires(IntegTestPreconditions.JavaHomeWithDifferentVersionAvailable)
+    def "Given disabled auto-detection When using daemon toolchain Then option is ignored resolving with expected toolchain"() {
```

Comment:
I added [here](https://github.com/gradle/gradle/pull/27898#discussion_r1481318879) more context the reason why `auto-detection` property isn't recognized by the Daemon toolchain , but as a summary has to do with having multiple source of truth about the Daemon toolchain definitions that was considered to be discussed in future iterations since is quite confusing having `gradle.properties` and `gradle-build.properties` if reusing the existing ones